docs/integrations/sources/surveymonkey.md

```diff
@@ -1,48 +1,64 @@
 
 :::
 
 <!-- env:oss -->
-## Prerequisites
+## Know before you begin
 
-**For Airbyte Open Source:**
-
-* Access Token
+You need a SurveyMonkey access token if you're using Airbyte Open Source to set up the connector. Use this token to authenticate your SurveyMonkey account with Airbyte Open Source.
 <!-- /env:oss -->
 
-## Setup guide
+## Set up the SurveyMonkey connector in Airbyte
+
 ### Step 1: Set up SurveyMonkey
-Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started). Register your application [here](https://developer.surveymonkey.com/apps/) Then go to Settings and copy your access token
+
+1. Create or log in to your [SurveyMonkey account](https://developer.surveymonkey.com/apps/).
+2. In My Apps, click **Add a New App**. The App Creation screen displays.
+3. Enter an **App Nickname** and **App Creator** email address.
+4. Select a public or private app type depending on your use case.
+5. Click **Create App**. The app overview page displays.
+6. Click **Settings**.
+7. In the Credentials section, copy your access token. You will use this token later in the guide.
+
+**Step result:** You registered a new SurveyMonkey app and access token.
+For more information about SurveyMonkey apps, read their [docs](https://developer.surveymonkey.com/api/v3/#SurveyMonkey-Api).
 
 ### Step 2: Set up the source connector in Airbyte
 
 <!-- env:cloud -->
 **For Airbyte Cloud:**
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. lick `Authenticate your account`.
-5. Log in and Authorize to the SurveyMonkey account
-6. Choose required Start date
-7. click `Set up source`.
+1. Log into your [Airbyte Cloud account](https://cloud.airbyte.com/workspaces).
+2. Select an existing workspace or create a new one by clicking **+ New Workspace**.
+3. In the navigation menu, click **Sources**. A new page displays all available Airbyte sources.
+4. Find and select the **SurveyMonkey** source. The Create a source page displays.
+5. For **Source name**, enter a name for this connector.
+6. Click **Authenticate your SurveyMonkey account**.
+7. Log in to Airbyte Cloud and authorize the SurveyMonkey account.
+8. Choose the required **Start date**.
+9. Click **Set up source**.
+
+Step result: You created a source connector in Airbyte Cloud.
 <!-- /env:cloud -->
 
 <!-- env:oss -->
 **For Airbyte Open Source:**
 
-1. Go to local Airbyte page.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. Add **Access Token**
-5. Choose required Start date
-6. Click `Set up source`.
+1. Go to your local Airbyte page.
```

Comment:
1. Unfortunately, I have no idea how to get to step one to test this use case. Maybe something more descriptive can help me find the right webpage.

2. The rest I copied from the Airbyte Cloud steps.